modules/salt/pages/formula-tftpd.adoc

```diff
@@ -1,7 +1,7 @@
 [[tftpd-formula]]
 = TFTPD Formula
 
-The TFTPD formula is an {smr} formula, used to configure the TFTP service on the branch server.
+The TFTPD formula is an {productname} {smr} formula, used to configure the TFTP service on the branch server.
```

Comment:
Sometimes we write:
`a {productname} {smr}`
or:
`an {productname} {smr}`

The former is suitable for Uyuni, the latter for Suse Manager.
Could we handle this `a` VS `an` systematically?

Reply:
This happens throughout the docs with {productname}, and I think the adoc ifdef is not fine-grained enough to handle it neatly. I think for retail we could possibly create an additional entity, so we would have one to use with {productname} to create "{productname} for Retail" and one that is just "Retail" for where we want something shorter (or use SMR? is that an approved abbreviation? Might need input from Pau).